src/Compute/Compute/help/New-AzDiskPurchasePlanConfig.md

```diff
@@ -0,0 +1,127 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
+online version: https://docs.microsoft.com/powershell/module/az.compute/new-azdiskpurchaseplanconfig.md
+schema: 2.0.0
+---
+
+# New-AzDiskPurchasePlanConfig
+
+## SYNOPSIS
+Creates a PurchasePlan Object
+
+## SYNTAX
+
+```
+New-AzDiskPurchasePlanConfig [-Publisher <String>] [-Name <String>] [-Product <String>]
```

Comment:
please update the help file since Publisher, Name and Product are required.

Reply:
Updated, Thanks